bids/layout/tests/test_path_building.py

```diff
@@ -13,6 +13,9 @@ def layout():
     data_dir = join(get_test_data_path(), '7t_trt')
     return BIDSLayout(data_dir)
 
+def test_path_building_pet(layout):
```

Comment:
adding a regression test: maybe an overkill?
will become redundant once a pet example include the tracer entity

Reply:
Maybe just note the condition on which it can be removed?